foreman_api_client.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_runtime_dependency "apipie-bindings", "= 0.0.15"
-  spec.add_runtime_dependency "rest-client",     ">= 2.0.0.rc1"
+  spec.add_runtime_dependency "rest-client",     ">= 2.0"
```

Comment:
This should probably be locked down a little bit.  What do you think of `"~> 2.0"`?